db/migrate/20170824152141_create_students.rb

```diff
@@ -0,0 +1,13 @@
+class CreateStudents < ActiveRecord::Migration[5.1]
+  def change
+    create_table :students do |t|
+      t.integer :admission_year, limit: 4
+      t.string :admission_no
+      t.date :regis_date
```

Comment:
The Rails convention for date stamp is `**_at`. So this should probably be `registered_at`.